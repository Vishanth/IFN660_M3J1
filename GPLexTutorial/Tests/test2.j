public class HelloWorld 
{
    public static void main(string[] args) 
    {
        int x;
		x=42;
    }
}