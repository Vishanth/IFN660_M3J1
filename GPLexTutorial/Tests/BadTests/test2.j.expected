FLOATLITERAL (3.14)
EOF